src/pages/RFOX/components/Overview/StakingInfo.tsx

```diff
@@ -106,7 +106,7 @@ export const StakingInfo: React.FC<StakingInfoProps> = ({
 
   const handleGetAssetClick = useCallback(
     (assetId: AssetId) => () => {
-      navigate(`/trade/${assetId}`)
+      navigate(`/trade/${assetId}/${arbitrumAssetId}/0`)
```

Comment:
Sell asset and amount are required for the routing to work following latest work in routing - two birds one stone, since the issue this fixes was about the default sell asset not making sense as a route